### ISSUE TITLE
Excessive eye contact

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#8027]
**Preferred term label:**
Excessive eye contact

**Synonyms**

**Definition (free text, please give PubMed ID)**
_will be discussed at next workshop_

**Parent term (use hpo.jax.org/app)**
Abnormal eye contact (#8024) 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-4422-1708

**Comment by pnrobinson** [comment]
added -- definition to be added later
